src/backend/base/langflow/logging/logger.py

```diff
@@ -230,8 +231,21 @@ def configure(
 
         if log_format is None or not is_valid_log_format(log_format):
             log_format = DEFAULT_LOG_FORMAT
-
-        logger.add(sys.stdout, level=log_level.upper(), format=log_format, backtrace=True, diagnose=True)
+        # pretty print to rich stdout development-friendly but poor performance, It's better for debugger.
+        # suggest directly print to stdout in production
+        log_stdout_pretty = os.getenv("LANGFLOW_LOG_STDOUT_PRETTY", "true") == "true"
```

Comment:
What about `LAGFLOW_PRETTY_LOGS`? 

I think if you update this we can merge it.